dataikuapi/dss/dataset.py

```diff
@@ -737,6 +737,25 @@ def add_time_partitioning_dimension(self, dim_name, period="DAY"):
     def add_raw_schema_column(self, column):
         self.settings["schema"]["columns"].append(column)
 
+    @property
+    def is_feature_group(self):
+        """
+        Indicates whether the Dataset is defined as a Feature Group, available in the Feature Store.
+
+        :rtype: bool
+        """
+        return self.settings["featureGroup"]
+
+    def set_feature_group(self, status):
```

Comment:
Rather make it a @property.setter

Reply:
I initially used a setter, but the idea was to allow specifying other params if the featureGroup has more options... But ok. :-)

Reply:
So, we eventually decided to leave it as is.